Update tag for RecoTracker-MkFit to V00-07-00

cmsbuild opened the pull request:
Move RecoTracker-MkFit data to new tag, see 
https://github.com/cms-data/RecoTracker-MkFit/pull/7

cmsbuild commented:
A new Pull Request was created by  @cmsbuild  for branch IB/CMSSW_12_2_X/master.

@cmsbuild, @smuzaffar, @iarspider, @ddaina can you please review it and eventually sign? Thanks.
@perrotta, @dpiparo, @qliphy you are the release manager for this.
cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

smuzaffar commented:
this has been tested via cms-data/RecoTracker-MkFit#7 , so no need to re-run tests here. It should be merged along with 
- https://github.com/cms-sw/cmsdist/pull/7427 
- https://github.com/cms-sw/cmssw/pull/35974

qliphy commented:
+1

qliphy commented:
merge

cmsbuild commented:
+1

**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22a650/20452/summary.html
**COMMIT**: 72085b46d0dc953798f9a00f2919f8715e8c8e83
**CMSSW**: CMSSW_12_2_X_2021-11-10-2300/slc7_amd64_gcc900
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmsdist/7443/20452/install.sh` to create a dev area with all the needed externals and cmssw changes.



The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
 - @mmasciov cms-sw/cmsdist#7427
 - @cms-sw cms-sw/cmsdist#7443

You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22a650/20452/git-recent-commits.json
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-22a650/20452/git-merge-result


## Comparison Summary



**Summary**:
 - No significant changes to the logs found
 - Reco comparison results: 23174 differences found in the comparisons
 - DQMHistoTests: Total files compared: 42
 - DQMHistoTests: Total histograms compared: 2901890
 - DQMHistoTests: Total failures: 33546
 - DQMHistoTests: Total nulls: 6
 - DQMHistoTests: Total successes: 2868316
 - DQMHistoTests: Total skipped: 22
 - DQMHistoTests: Total Missing objects: 0
 - DQMHistoSizes: Histogram memory added: -0.086 KiB( 41 files compared)
 - DQMHistoSizes: changed ( 136.793 ): -0.031 KiB       JetMET/SUSYDQM
 - DQMHistoSizes: changed ( 136.874 ): -0.055 KiB       JetMET/SUSYDQM
 - Checked 177 log files, 37 edm output root files, 42 DQM output files
 - [TriggerResults](https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_12_2_X_2021-11-10-2300+22a650/46792/triggerResults): no differences found
